tracecat/api/routers/public/webhooks.py

```diff
@@ -43,3 +43,36 @@ async def incoming_webhook(
         enable_runtime_tests=enable_runtime_tests,
     )
     return response
+
+
+@router.post("/{path}/{secret}/wait", tags=["public"])
+async def incoming_webhook_sync(
```

Comment:
could we rename this to `incoming_webhook_wait`? `*_sync` might be slightly misleading implying that this function is a non-async function

Reply:
> could we rename this to `incoming_webhook_wait`? `*_sync` might be slightly misleading implying that this function is a non-async function

done